tools/python/FindOptimizerOpsetVersionUpdatesRequired.py

```diff
@@ -59,18 +62,53 @@ def get_call_args_from_file(filename, function_or_declaration):
     return results
 
 
+def get_multiline_call_args_from_file(filename, function_or_declaration):
```

Comment:
```suggestion
def get_multiline_call_args_from_file(filename: str, function_or_declaration: str) -> List[str]:
```

Helpful to annotate the type